src/main/drivers/barometer/barometer_qmp6988.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define QMP6988_I2C_ADDR                      (0x70)
+#define QMP6988_DEFAULT_CHIP_ID               (0x5c)
+
+#define QMP6988_CHIP_ID_REG                   (0xD1)  /* Chip ID Register */
```

Comment:
Are these actually needed outside of the driver? If not they should go into the .c file.